src/View/StringTemplate.php

```diff
@@ -284,7 +306,7 @@ protected function _formatAttribute($key, $value, $escape = true)
         $truthy = [1, '1', true, 'true', $key];
         $isMinimized = isset($this->_compactAttributes[$key]);
         if ($isMinimized && in_array($value, $truthy, true)) {
-            return "$key=\"$key\"";
+            return "$key";
```

Comment:
Short form attributes also break in xhtml doc types. This change would make using the helpers impossible in xhtml doc types.